src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,796 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { act, fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import { Slider, focusThumb } from './Slider';
+
+const createBoundingClientRect = (
+  left: number,
+  top: number,
+  right: number,
+  bottom: number,
+): DOMRect => ({
+  left,
+  top,
+  right,
+  bottom,
+  width: right - left,
+  height: bottom - top,
+  x: left,
+  y: top,
+  toJSON: () => '',
+});
+
+/**
+ * Setup default size for slider container to be used by all slider tests
+ */
+const getBoundingClientRect = Element.prototype.getBoundingClientRect;
+const sliderContainerSize = createBoundingClientRect(10, 0, 1010, 60);
+Element.prototype.getBoundingClientRect = () => sliderContainerSize;
+
+afterAll(() => {
+  Element.prototype.getBoundingClientRect = getBoundingClientRect;
+});
+
+const defaultSingleValue = [50];
+
+const assertBaseElement = (container: HTMLElement) => {
```

Comment:
This function doesn't really help with anything. Maybe add more common logic here? Or remove it.

Reply:
added some more selectors